### DISCUSSION TITLE
Server side rendering for nuxt

### DISCUSSION THREAD

**Discussion by jessielaf** [vueform/vueform#109] [Ideas]
Hi,

I really like the work you put into vueform. I am using Nuxt and it is really helpful that there is a nuxt module. However, in the guides, but also in the nuxt module it is apparent that VueForm is only meant to be run client side. This does give a delay in the forms to be loaded. For this reason I cannot yet implement vueform in my company since interactivity on load is very important to us.

My question is if it is possible to make the forms render on server side? Can I help in order to enable vueform to be rendered on server side?

**Comment by adamberecz** [comment]
Hi @jessielaf - yes, the thing is that unfortunately Vueform is not SSR friendly at the moment. We aim to change this in the future and any help with relevant experience is appreciated! 💚

**Reply by dkovach** [reply]
I take it this is still the case?

And where are guides for succinct use with Nuxt to force it to work client side? I am struggling with it when it comes to using it as a component w the mixin to mimic vueForm...
